libraries/AP_Math/AP_Math.cpp

```diff
@@ -399,9 +399,13 @@ Vector3F get_vel_correction_for_sensor_offset(const Vector3F &sensor_offset_bf,
  */
 float calc_lowpass_alpha_dt(float dt, float cutoff_freq)
 {
-    if (dt <= 0.0f || cutoff_freq <= 0.0f) {
+    if (is_negative(dt) || !is_positive(cutoff_freq)) {
+        INTERNAL_ERROR(AP_InternalError::error_t::flow_of_control);
```

Comment:
I think we should use, "AP_InternalError::error_t::invalid_arg_or_result".  The result won't be any different but this might be slightly closer to describing what is wrong.